specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -824,6 +825,10 @@
         "maxConcurrentConnections": {
           "type": "object",
           "description": "The maximum concurrent connection count for the source data store. Type: integer (or Expression with resultType integer)."
+        },
+        "disableMetricsCollection": {
+          "type": "object",
+          "description": "Specifies whether to disable collecting data source metrics. Type: boolean (or Expression with resultType boolean)."
```

Comment:
Can you update x-ms-examples for the operation that uses this model in the request body or response body and populate the example with this new property that has been added in this PR.